lapack/netlib/generate_errors.go

```diff
@@ -0,0 +1,57 @@
+// Copyright ©2018 The Gonum Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+// +build ignore
+
+package main
+
+import (
+	"fmt"
+	"go/parser"
+	"go/printer"
+	"go/token"
+	"log"
+	"os"
+	"path/filepath"
+)
+
+const errorFile = "../../../gonum/lapack/gonum/errors.go"
+
+func main() {
+	path, err := filepath.Abs(errorFile)
+	if err != nil {
+		log.Fatalf("no absolute path for %q: %v", errorFile, err)
+	}
+
+	fset := token.NewFileSet()
+	f, err := parser.ParseFile(fset, path, nil, 0)
+	if err != nil {
+		log.Fatalf("failed to parse %q: %v", path, err)
+	}
+
+	dst := filepath.Base(errorFile)
+	o, err := os.Create(dst)
+	if err != nil {
+		log.Fatalf("failed to create %q: %v", dst, err)
+	}
+	defer o.Close()
+
+	fmt.Fprintln(o, header)
+	p := printer.Config{
+		Mode:     printer.UseSpaces | printer.TabIndent,
+		Tabwidth: 8,
+	}
+	p.Fprint(o, fset, f.Decls)
+	fmt.Fprintln(o)
+}
+
+const header = `// Code generated by "go generate gonum.org/v1/netlib/lapack/netlib”; DO NOT EDIT.
+
+// Copyright ©2018 The Gonum Authors. All rights reserved.
```

Comment:
Should the year be 2015, the same as before?

Reply:
I can get it to grab the first comment block and reflect that here.